Structs/__init__.py

```diff
@@ -186,6 +186,18 @@ def convert_fstruct(
             )
 
         return expected_type(fstruct)  # type: ignore
+    
+    def convert_arg(
+        arg: Any,
+        idx: int
+    ) -> Any:
+        if isinstance(arg, unrealsdk.FArray):
+            return [convert_arg(a, i) for i, a in enumerate(arg)]
```

Comment:
This isn't going to work, the index is the index of the field on the struct, array entries aren't in the struct.

Reply:
Resolved in latest commit